index.js

```diff
@@ -32,7 +32,7 @@ function sendBeacon(url, payload) {
 
   const evalString = send.toString() + invokeString
 
-  const subprocess = spawn(process.argv[0], ['-e', evalString], {
+  const subprocess = spawn("node", ['-e', evalString], {
```

Comment:
I think no need for this change?

Reply:
Problem with _process.argv[0]_ is that if initial launch command is _ts-node_ instead of _node_, code breaks. Alternatively, something like this can be done:

```javascript
let subprocess = null;
try {
    subprocess = spawn(process.argv[0], ...);
} catch (e) {
    subprocess = spawn("node", ...);
}
```

Reply:
good point, let's use "node"